### PR TITLE
Hide passwords in edit/add form

### PR DESCRIPTION
**Pull request by BenjaminHae**
@Pofilo Could you please take a look at it? I've merged the add and edit forms, so there is some potential for errors.

Implements #222

### CODE REVIEW

File: src/password.php [hunk 1]
```diff
@@ -168,23 +142,24 @@
...
                     <input class="form-control" id="edititeminput" type="text" />
                 </div>
                 <div class="form-group">
-                    <label for="edititeminputpw" class="control-label">Password:</label>
+                    <label for="edititeminputpw" class="control-label">Password</label>
                     <div class="input-group">
-                        <input class="form-control passwordText" id="edititeminputpw" type="text" autocomplete="off" placeholder="Leave blank to generate one"/>
+                        <input class="form-control passwordText" id="edititeminputpw" type="password" autocomplete="off" placeholder="Leave blank to generate one"/>
                         <span class="input-group-btn">
                             <button id="editPasswordInput" class="btn btn-warning" type="button" title="generate new password"><i class="glyphicon glyphicon-refresh"></i></button>
```
Comment (Pofilo) [inline src/password.php]: Generate new password

File: src/password.php [hunk 2]
```diff
@@ -168,23 +142,24 @@
...
                             <button class="btn btn-default" type="button" id="editAccountShowPassword" title="show current password"><i class="glyphicon glyphicon-eye-open"></i></button>
```
Comment (Pofilo) [inline src/password.php]: Show current password

**Comment by Pofilo** [comment]
I just made 2 comments on typo but that's all ^^

Honnestly, changes are really good, I've try every possibilities to see any bug, I didn't find any. Plus, the implementation looks good to me.

The way you did it works and is really user-friendly with the little popup on *Dismiss* to get back the old password.

File: src/password.php (hunk 2 above)
Comment (BenjaminHae) [reply src/password.php]: Is this the same as the one above or did you mean another line?

File: src/password.php (hunk 1 above)
Comment (BenjaminHae) [reply src/password.php]: I've changed it to "Leave blank to get a random password"

File: src/password.php (hunk 2 above)
Comment (Pofilo) [reply src/password.php]: Oh sorry, by typo I ment the capital letter at the beginning !
You know, just to say something because the rest was perfect ^^

Comment (BenjaminHae) [reply src/password.php]: Thanks :)

Comment (BenjaminHae) [reply src/password.php]: I got confused by the highlighted lines, sorry. Now I know what you meant.

**Pull request merged**